### PR TITLE
Add new endpoint, update documentation and version accordingly

### PR DESCRIPTION
**Pull request by Kadaaran**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Change log
 
+## 0.4.0
+* Add support for returning featured collection.
```
Comment (dvandersluis) [inline CHANGES.md]:
```suggestion
* Add support for returning featured collections.
```

File: CHANGES.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
...
+* Add support for collection resource endpoint.
```
Comment (dvandersluis) [inline CHANGES.md]: I'm not sure what this means?

Comment (Kadaaran) [reply CHANGES.md]: English issue here I believe, to me I thought _add support_ was the equivalent for _add new endoint_ 
I can add `Add new endpoint for `Collections` resource ...` ?

File: CHANGES.md (hunk 1 above)
Comment (Kadaaran) [reply CHANGES.md]: Ah good catch!

File: CHANGES.md (hunk 2 above)
Comment (Kadaaran) [reply CHANGES.md]: Fixed! 👍

**Review by dvandersluis — APPROVED** [review]

**Pull request merged**